### PR TITLE
Skip previously bundled extensions

### LINKED ISSUE #3396: [EXTENSIONS-LOADER]: can't load renderer for "example-extension": TypeError: Class extends value ...
Sentry Issue: [K8SLENS-BT](https://sentry.io/organizations/k8slens/issues/2514446191/?referrer=github_integration)

```
[EXTENSIONS-LOADER]: can't load renderer for "example-extension": TypeError: Class extends value undefined is not a constructor or null
    at Object.<anonymous> (C:\Users\ntshoe_i\AppData\Local\Programs\kontena-lens\resources\extensions\example-exte...
```

### PR DESCRIPTION
**Pull request by jakolehm**
Fixes #3396

### CODE REVIEW

**Review by aleksfront — APPROVED** [review]

**Comment by Nokel81** [comment]
Alternative is https://github.com/lensapp/lens/pull/3397 but I think this PR probably will work better.

File: src/extensions/extension-compatibility.ts [hunk 1]
```diff
@@ -31,3 +31,7 @@ export function isCompatibleExtension(manifest: LensExtensionManifest): boolean
 
   return false;
 }
+
+export function isCompatibleBundledExtension(manifest: LensExtensionManifest): boolean {
+  return manifest.version === appSemVer.raw;
```
Comment (jim-docker) [inline src/extensions/extension-compatibility.ts:36]: Does this work when `appSemVer` has a prerelease tag? (Does `manifest.version` have the matching tag, at least for bundled v5 extensions?)

Comment (Nokel81) [reply src/extensions/extension-compatibility.ts:36]: Yes do (I believe). We set them to the exact version that the `package.json` was changed to

Comment (jakolehm) [reply src/extensions/extension-compatibility.ts:36]: During build the app version is synced to the bundled extensions: https://github.com/lensapp/lens/blob/master/build/set_build_version.ts#L45

**Comment by Nokel81** [comment]
closes #3397

**Pull request merged**